### ISSUE TITLE
Add the algorithms model which is not added in dataset

### ISSUE THREAD

**Issue by ayushrathore111** [Rakesh9100/ML-Project-Drug-Review-Dataset#82]
### Prerequisites

- [X] I am an Open Source Contributor
- [X] I checked to make sure that this issue has not already been filed

### Description

 Hi @Rakesh9100  want to add these algorithms model which is not added to your dataset kindly assign to me so that I can make pull requests...
1. MPNN(multilayer perceptron neural network) 2. GMDH 3. GENETIC PROGRAMMING 4. SVM 5. KNN 6. K-MEANS 7. K-NEAREST NEIGHBOR 8. DECISION TREE 9. NAIVE BAYES 10. RANDOM FOREST 11. ADABOOST 12. GRADIENT BOOSTING 13. XGBOOST 14. LIGHTGBM 15. CATBOOST 16. RNN 17. LSTM 18. GRU 19. CNN 20. CNN-LSTM 21. CNN-GRU 22. CNN-RNN 23. CNN-CNN 24. CNN-CNN-LSTM 25. CNN-CNN-GRU 26. Gaussian mixture models 26. Gaussian naive bayes 27. Fussy logic system 28.Q-learning,deep-q networks

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/ML-Project-Drug-Review-Dataset/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.

**Comment by Rakesh9100** [comment]
@ayushrathore111 Change the title to 2-3 words

**Comment by Rakesh9100** [comment]
@ayushrathore111 Change the title to 4-5 words

**Comment by ayushrathore111** [comment]
@Rakesh9100 updated the title
thank you
